yarn-project/noir-contracts/src/contracts/noir-aztec3/src/abi.nr

```diff
@@ -4,9 +4,9 @@ global MAX_NEW_COMMITMENTS: comptime Field = 4;
 global MAX_NEW_NULLIFIERS: comptime Field = 4;
 global MAX_PRIVATE_CALL_STACK : comptime Field = 4;
 global MAX_PUBLIC_CALL_STACK: comptime Field = 4;
-global MAX_L1_MSG_STACK : comptime Field = 2;
+global MAX_L2_TO_L1_MSG_STACK : comptime Field = 2;
+global PUBLIC_INPUTS_LENGTH : comptime Field = 41;
```

Comment:
Why is this jumping 4 but seems like we are only including 2 extra?

Reply:
ah, kernel length was removed in a master merge, messed up numbering